devtools-html/debugger.html
# [WhyPaused] Move why paused to the call stack

Associated Issue: #3111 

### Summary of Changes

* Moved WhyPaused component to the end of the call stack
* Changed formatting to a more subtle one
* Centered the text

I definitely don't like the color of text in dark theme (it looks too visible), but I need your feedback on this.

### Test Plan

- Open any page in debugger and set a breakpoint
- make debugger break code execution on breakpoint
- See the WhyPaused component to appear at the end of the call stack

### Screenshots/Videos (OPTIONAL)
|Dark theme|Light theme|Firebug theme|
|---|---|---|
|![image](https://user-images.githubusercontent.com/69977/26877252-07df0bc2-4bd5-11e7-89db-c1e23ae9611d.png)|![image](https://user-images.githubusercontent.com/69977/26877322-42ae01b8-4bd5-11e7-8529-c7f2f3bed5da.png)|![image](https://user-images.githubusercontent.com/69977/26877392-83331570-4bd5-11e7-903a-60a9875ed0bd.png)|
